om4labs/m6plot/xyplot.py

```diff
@@ -72,6 +74,8 @@ def xyplot(
     # set visual backend
     if interactive is False:
         plt.switch_backend("Agg")
```

Comment:
this is probably machine specific

Reply:
Agg is a headless backend.  I'm pretty confident it is universal.